changelogs/fragments/9123-redfish-command-custom-oem-params.yml

```
@@ -0,0 +1,2 @@
+minor_changes:
+  - redfish_command - add ``update_custom_oem_header`` and ``update_custom_oem_params`` options (https://github.com/ansible-collections/community.general/pull/9123).
```

There are actually three new options :)
```suggestion
  - redfish_command - add ``update_custom_oem_header``, ``update_custom_oem_params``, and ``update_custom_oem_mime_type`` options (https://github.com/ansible-collections/community.general/pull/9123).
```

---

You’re right. I forgot to add it to the changelog after I implemented the MIME type option 😄